app/models/legacy-base.model.js

```diff
@@ -65,6 +65,159 @@ class LegacyBaseModel extends BaseModel {
   static get schema () {
     throw new Error('defaultSchema() not implemented in child class')
   }
+
+  /**
+   * Translations for non-standard column names to our standardised ones
+   *
+   * The primary example of non-standard column names in the legacy data is timestamps. For example, we have found the
+   * column which has the timestamp for when a record is created has been named `date_created`, 'created', 'modified',
+   * and 'created_at`.
+   *
+   * Within our code we want to just be using `created_at`, in this example, as this is what the field would have been
+   * called if Knex had been used to generate the migrations and build the tables.
```

Comment:
As discussed we probably want to distinguish between what we would want to see the column as (ie. `created_at`) and how we want to refer to it in our code (ie. `createdAt`)